### ISSUE TITLE
Getting a 500 when searching for something the first time

### ISSUE THREAD

**Issue by arnoldnipper** [peeringdb/peeringdb#1009]
**Describe the bug**
When searching for an ASN I get a 500 the first time. Hitting refresh gives the right result.

**To Reproduce**
1. Go to https://peeringdb.com/
2. Search for an ASN (e.g. 63264)
3. See error
4. Hit refresh and get result

**Expected behavior**
getting the result at the first search

**Who is affected by the problem?**
at least me (encountered this behaviour 3x within a week)

**What is the impact?**
users are at least confused

**Are there security concerns?**
no

**Are there privacy concerns?**
no

**What are the proposed actions?**

**What is the proposed priority?**
urgent

![Screenshot 2021-07-22 20 18 45](https://user-images.githubusercontent.com/16888757/126689959-3f0cbcb5-e4c7-471f-87b4-7f8c572ef39d.png)

**Comment by leovegoda** [comment]
I also saw this earlier today but I was searching for a name and not an ASN

**Comment by vegu** [comment]
This appears to sometimes happen when a search occurs while the re-indexer runs (which currently runs once per minute)

**Comment by mcmanuss8** [comment]
We should fix this ASAP. It's not a good user experience

**Comment by vegu** [comment]
We have been unable to reproduce this issue on dev or beta at all. 

It **still happens** on production now that we have rolled out 2.29.

The issue appears to be some sort of race condition when the search index is being updated while a search request is ongoing. Cause and possible solutions are being looked into - for the time being we have increased the delay of the update index job from 1 minute to 5 minutes to cut down the frequency of this error.
